build/root/usr/local/bin/ci_entrypoint_custom.sh

```diff
@@ -1,11 +1,34 @@
 #! /usr/bin/env bash
 
+set -x
 set -o pipefail
 set -o errexit
 set -o nounset
 
-# This file intentionally left blank.
+PSAP_SECRET_PATH=/var/run/psap-entitlement-secret
 
-# This file will run when you comment "/test custom" on your PR.
-# Feel free to modify this file to run whatever you want in order to test your PR
-# Please restore the file to this original state once you're done
+source ci_entrypoint_gpu-operator.sh source
+
+prepare_cluster_for_gpu_operator
+
+gpu_node_hostname=$(oc get nodes -oname -lnode.kubernetes.io/instance-type=g4dn.xlarge -ojsonpath={.items[].metadata.labels} | jq -r '.["kubernetes.io/hostname"]')
+
+if [ -z "$gpu_node_hostname" ]; then
+    echo "Couldn't find the GPU node ..."
+    oc get nodes --show-labels | sed 's|,|,- |g' | tr ',' '\n'
```

Comment:
The output will be messy on SNO clusters for example, where you have dual (master,worker) roles separated with `','`.  If that's not a problem to you, happy to `/lgtm` again.

Reply:
good point :)
this file `ci_entrypoint_custom.sh` will be reverted once the PR is merged, so not an issue here